### PR TITLE
feat(Drawer): subtitle and global classes

### PR DESCRIPTION
**Pull request by talend-glorieux**
**What is the problem this PR is trying to solve?**
* Can't override Drawer style
* Can't add subtitle to Drawer header

**What is the chosen solution to this problem?**
* Adds global classes
* Adds subtitle property

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/Drawer/Drawer.component.js [hunk 1]
```diff
@@ -59,12 +57,11 @@ function DrawerContainer({ stacked, className, children, withTransition = true,
...
 			stacked,
-		});
+		},
+	);
 	return (
 		<div className={drawerContainerClasses} {...rest}>
-			<div className={theme['tc-drawer-container']}>
-				{children}
-			</div>
+			<div className={theme['tc-drawer-container']}>{children}</div>
```
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: global class could be added?

```jsx
<div className={theme['tc-drawer-container'], 'tc-drawer-container'}>{children}</div>
```

File: packages/components/src/Drawer/Drawer.component.js [hunk 2]
```diff
@@ -80,25 +77,36 @@ export function cancelActionComponent(onCancelAction) {
...
+	);
 	return <Action className={theme['tc-drawer-close-action']} {...enhancedCancelAction} />;
 }
 
-function DrawerTitle({ title, children, onCancelAction }) {
+export function subtitleComponent(subtitle) {
+	if (!subtitle || subtitle === '') {
+		return null;
+	}
+	return <h2 title={subtitle}>{subtitle}</h2>;
```
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: :clap:

File: packages/components/src/Drawer/Drawer.component.js [hunk 3]
```diff
@@ -80,25 +77,36 @@ export function cancelActionComponent(onCancelAction) {
...
+}
+
+function DrawerTitle({ title, subtitle, children, onCancelAction }) {
 	if (!title) {
 		return null;
 	}
 	return (
-		<div className={theme['tc-drawer-header']}>
-			<div className={theme['tc-drawer-header-title']}>
+		<div className={`tc-drawer-header ${theme['tc-drawer-header']}`}>
```
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: Why not using classNames instead?

File: packages/components/src/Drawer/Drawer.component.js [hunk 4]
```diff
@@ -124,11 +133,7 @@ DrawerContent.propTypes = {
 };
 
 function DrawerFooter({ children }) {
-	return (
-		<div className={theme['tc-drawer-footer']}>
-			{children}
-		</div>
-	);
+	return <div className={`tc-drawer-footer ${theme['tc-drawer-footer']}`}>{children}</div>;
```
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: Could you change element by `footer` ? 
(I know it's not the purpose of the pull-request)

File: packages/components/src/Drawer/Drawer.component.js (hunk 3 above)
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: Could you change element by `header` ?
(I know it's not the purpose of the pull-request)

File: packages/components/src/Drawer/Drawer.component.js [hunk 5]
```diff
@@ -80,25 +77,36 @@ export function cancelActionComponent(onCancelAction) {
...
+			link: true,
+		},
+		onCancelAction,
+	);
 	return <Action className={theme['tc-drawer-close-action']} {...enhancedCancelAction} />;
 }
 
-function DrawerTitle({ title, children, onCancelAction }) {
+export function subtitleComponent(subtitle) {
+	if (!subtitle || subtitle === '') {
```
Comment (frassinier) [inline packages/components/src/Drawer/Drawer.component.js]: `if (!subtitle || !subtitle.length) {` ?

File: packages/components/src/Drawer/Drawer.component.js (hunk 3 above)
Comment (talend-glorieux) [reply packages/components/src/Drawer/Drawer.component.js]: I don't need any conditional class names. Plus I would have to use it for every global class change I made.

**Pull request merged**